src/components/Checkbox/Checkbox.module.css

```diff
@@ -63,6 +63,16 @@
   @apply font-medium;
 }
 
+.sbui-checkbox__label-container__label__span .sbui-checkbox__label-text-before,
+.sbui-checkbox__label-container__label__span .sbui-checkbox__label-text-after {
+  @apply text-gray-400 dark:text-gray-300;
+}
+
+.sbui-checkbox__label-text-before::after,
+.sbui-checkbox__label-text-after::before {
+  content: ' : ';
```

Comment:
@jonthomp ah, sorry, I don't think we need these - my bad example in the issue description.
I think I was just using an example from the supabase dashboard but the actual ui library doesn't need to add any `:` characters I don't think

Reply:
@MildTomato no problemo, all removed